api-contract/src/main/java/uk/gov/hmcts/payment/api/contract/TelephonyRequest.java

```diff
@@ -0,0 +1,54 @@
+package uk.gov.hmcts.payment.api.contract;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.experimental.Wither;
+import uk.gov.hmcts.payment.api.contract.util.CurrencyCode;
+import uk.gov.hmcts.payment.api.contract.util.Service;
+
+import javax.validation.constraints.*;
+import java.math.BigDecimal;
+
+import static com.fasterxml.jackson.annotation.JsonInclude.Include.NON_NULL;
+
+@JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)
+@JsonInclude(NON_NULL)
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder(builderMethodName = "telephonyRequestDtoWith")
+@Wither
+public class TelephonyRequest {
```

Comment:
I suggest changing DTO name to telephonyCardPaymentsRequest to represent the API

Reply:
Fixed.